index.html

```diff
@@ -1,57 +1,59 @@
 <head>
-    <title>Bitmovin Demo</title>
+    <title>Bitmovin SDK</title>
     <meta charset="UTF-8"/>
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
-    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
-    <!-- Bitmovin player -->
-    <script type="text/javascript" src="https://bitmovin-a.akamaihd.net/bitmovin-player/stable/7/bitmovinplayer.js"></script>
-    <!-- Bootstrap 3.3.7 -->
-    <link rel="stylesheet" href="css/bootstrap.min.css" crossorigin="anonymous">
-    <link rel="stylesheet" href="css/bitmovin-demo.css">
-
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Dosis" rel="stylesheet" type="text/css"/>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/style.css">
+    <!-- Bitmovin Player -->
+    <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
     <script type="text/javascript" src="js/main.js"></script>
+	  <script type="text/javascript" src="webOSTVjs-1.1.0/webOSTV.js"></script>
 </head>
 <body>
 <div id="wrapper">
-
     <div id="banner">
-        <div class="logo"><a href="https://bitmovin.com"><img src="images/bitmovin-logo.png"></a></div>
-        <div class="title"><h1 class="bitmovin-headline">Bitmovin Player Demo</h1></div>
+        <div class="logo"><a href="https://bitmovin.com"><img src="https://ox4zindgwb3p1qdp2lznn7zb-wpengine.netdna-ssl.com/wp-content/themes/Bitmovin-V-0.1/images/logo4.png"></a></div>
+        <div class="title"><h1 class="bitmovin-headline">HTML5 Adaptive Streaming Player for MPEG-DASH & HLS.</h1></div>
         <div class="clearfix"></div>
     </div>
 
     <div class="container-fluid">
         <div class="content">
             <div class="row bitmovin-description">
-                <div class="col-lg-12 text-center" id="source-names">
-
-                </div>
+                <div class="col-lg-12 text-center">Your videos play everywhere with low startup delay, no buffering and in highest quality.</div>
+            </div>
+            <div id="webserver-warning">
+                <h2>Unsupported Protocol</h2>
+                <p>
+                    This file has been loaded using the unsupported "file" protocol. Please use a
+                    <a href="https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_is_a_web_server" target="_blank">web server</a>
+                    and open this page using http or https.
+                </p>
             </div>
             <div class="row">
                 <div class="col-md-offset-1 col-md-10 col-lg-offset-2 col-lg-8">
-                    <div id="player"></div>
+                    <div id="my-player"></div>
```

Comment:
why changing the id? `my-player` sounds odd.

Reply:
This is our getting started example, but I can change that, sure.